db/liquibase/changesets/20180915_collection.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="avram" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="ansingh" id="add_collection_table">
+        <createTable tableName="collection">
+            <column name="id" type="bigserial" autoIncrement="true">
```

Comment:
Does `autoIncrement="true"` have any effect along with `type="bigserial"`? I suspect `autoIncrement="true"` is redundant. Did you try leaving it out?